[2025-01-05 10:28:29,939] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:28:29,974] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:28:29,974] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:28:29,975] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-01-05 10:28:29,975] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:28:30,026] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2024-12-23 00:00:00+00:00
[2025-01-05 10:28:30,035] {standard_task_runner.py:52} INFO - Started process 14996 to run task
[2025-01-05 10:28:30,046] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data', 'cria_pasta', 'scheduled__2024-12-23T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/weather_data.py', '--cfg-path', '/tmp/tmpf8rshcc5', '--error-file', '/tmp/tmpc5ifljox']
[2025-01-05 10:28:30,050] {standard_task_runner.py:80} INFO - Job 48: Subtask cria_pasta
[2025-01-05 10:28:30,243] {task_command.py:370} INFO - Running <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [running]> on host daniel-virtual-machine
[2025-01-05 10:28:30,486] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_data
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2024-12-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-23T00:00:00+00:00
[2025-01-05 10:28:30,494] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-05 10:28:30,498] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/daniel/Documents/Alura/python_apache_airflow_pipeline/week=2024-12-30"']
[2025-01-05 10:28:30,530] {subprocess.py:85} INFO - Output:
[2025-01-05 10:28:30,547] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-05 10:28:30,659] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=cria_pasta, execution_date=20241223T000000, start_date=20250105T132829, end_date=20250105T132830
[2025-01-05 10:28:30,766] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-05 10:28:30,935] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-05 10:33:00,734] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:33:00,749] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:33:00,749] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:33:00,749] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-01-05 10:33:00,749] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:33:00,771] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2024-12-23 00:00:00+00:00
[2025-01-05 10:33:00,776] {standard_task_runner.py:52} INFO - Started process 16584 to run task
[2025-01-05 10:33:00,784] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data', 'cria_pasta', 'scheduled__2024-12-23T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/weather_data.py', '--cfg-path', '/tmp/tmpb7ywr8zo', '--error-file', '/tmp/tmp28uakpb6']
[2025-01-05 10:33:00,785] {standard_task_runner.py:80} INFO - Job 49: Subtask cria_pasta
[2025-01-05 10:33:00,865] {task_command.py:370} INFO - Running <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [running]> on host daniel-virtual-machine
[2025-01-05 10:33:00,922] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_data
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2024-12-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-23T00:00:00+00:00
[2025-01-05 10:33:00,923] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-05 10:33:00,924] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/daniel/Documents/Alura/python_apache_airflow_pipeline/week=2024-12-30"']
[2025-01-05 10:33:00,932] {subprocess.py:85} INFO - Output:
[2025-01-05 10:33:00,937] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-05 10:33:00,960] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=cria_pasta, execution_date=20241223T000000, start_date=20250105T133300, end_date=20250105T133300
[2025-01-05 10:33:00,999] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-05 10:33:01,029] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-05 10:57:49,211] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:57:49,228] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [queued]>
[2025-01-05 10:57:49,230] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:57:49,230] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-01-05 10:57:49,230] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-01-05 10:57:49,251] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2024-12-23 00:00:00+00:00
[2025-01-05 10:57:49,256] {standard_task_runner.py:52} INFO - Started process 4439 to run task
[2025-01-05 10:57:49,262] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data', 'cria_pasta', 'scheduled__2024-12-23T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/weather_data.py', '--cfg-path', '/tmp/tmp5xttwuft', '--error-file', '/tmp/tmpij250c9l']
[2025-01-05 10:57:49,268] {standard_task_runner.py:80} INFO - Job 59: Subtask cria_pasta
[2025-01-05 10:57:49,331] {task_command.py:370} INFO - Running <TaskInstance: weather_data.cria_pasta scheduled__2024-12-23T00:00:00+00:00 [running]> on host daniel-virtual-machine
[2025-01-05 10:57:49,395] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_data
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2024-12-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-23T00:00:00+00:00
[2025-01-05 10:57:49,400] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-05 10:57:49,401] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/daniel/Documents/Alura/python_apache_airflow_pipeline/week=2024-12-30"']
[2025-01-05 10:57:49,411] {subprocess.py:85} INFO - Output:
[2025-01-05 10:57:49,416] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-05 10:57:49,451] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=cria_pasta, execution_date=20241223T000000, start_date=20250105T135749, end_date=20250105T135749
[2025-01-05 10:57:49,480] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-05 10:57:49,512] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
